src/api/apiConfig.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export const api = axios.create({
+    baseURL: 'http://localhost:3000',
+});
```

Comment:
interceptors? 

Also, could you please add the following  logic: when your token not valid - run logout